docs/docs-beta/docs/dagster-plus/deployment/environment-variables/dagster-ui.md

```diff
@@ -5,4 +5,115 @@ sidebar_position: 1
+2. Click the **Trash icon** on the right of the environment variable you want to delete
+3. A modal will open to confirm that you want to delete the environment variable. Click **Yes, delete** to delete the environment variable, or **Cancel** to close the modal and not delete the environment variable.
+
+## How to view the value an environment variable
+To view the value of an environment variable:
+
+1. [Navigate to the environment variables page](#how-to-find-the-environment-variables-page)
+2. Click the **eye icon** in the **Value** column of the environment variable you want to view
+3. Click the **eye icon** again to hide the value.
+
+:::note
+Viewing an environment variable only reveals the value to you. It doesn't show the value in plaintext to all users. If you navigate away from the environment variables page or reload the page, the value will be hidden again.
+:::
+
+
+
+## How to add an environment variable for a Branch Deployment
+If you want to give branch deployments access to an existing environment variable:
+
+1. [Navigate to the environment variables page](#how-to-find-the-environment-variables-page)
+2. Click the **Edit** button on the right of the environment variable you want to edit
+3. A modal will open. Click the **Branch deployments** check-box to allow branch deployments to access the environment variable
+4. Click **Save**
+
+If you want to provide branch deployments a different value for the same environment variable key, see [How to set different values for environment variables in different deployment environments](#how-to-set-different-values-for-environment-variables-in-different-deployment-environments)
+
+
+
+## How to export environment variables locally
+You can export environment variables that have the **Local** scope selected into a `.env` file to use locally.
+
+To download the file:
+1. [Navigate to the environment variables page](#how-to-find-the-environment-variables-page)
+2. Click the **arrow menu** to the right of the **+ Add environment variable** button
+3. Click **Download local environment variables**. A file named `env.txt` will be downloaded
+
+To use the downloaded environment variables for local Dagster development:
+1. Rename the downloaded `env.txt` file to `.env`
+2. Move the file to the directory where you run `dagster dev` or `dagster-webserver`
+3. Run `dagster dev` or `dagster-webserver`
+4. Confirm that the environment variables have been loaded by locating a log message that begins with `Loaded environment variables from .env file`
+
+
+## How to set different values for environment variables in different deployment environments
+You can create multiple instances of the same environment variable key with different values. This allows you to provide different values to different deployment environments. For example, you may want to use different Snowflake credentials for your production deployment than in branch deployments.
```

Comment:
this section feels a bit wordy to me. i'm going to revisit it in an hour or so

Reply:
It honestly doesn't seem to bad to me